index.html

```diff
@@ -135,7 +176,7 @@
         Object.keys(workers).map(function (key) {
           var results = performance.getEntriesByType('measure')
             .filter(function (measure) {
-              return measure.name.indexOf(key) > -1;
+              return measure.name.indexOf(key) === 0;
```

Comment:
The old test was not strict enough: the `single-arraybuffer` results were pulled into the `arraybuffer` results.